packages/cactus-plugin-bungee-hermes/src/test/typescript/integration/bungee-merge-views.test.ts

```diff
@@ -0,0 +1,402 @@
+import {
+  IListenOptions,
+  LogLevelDesc,
+  LoggerProvider,
+  Secp256k1Keys,
+  Servers,
+} from "@hyperledger/cactus-common";
+import "jest-extended";
```

Comment:
can you please add tests with the privacy policies?

Reply:
The test tests one of them. should I add the other ?